command/base_helpers.go

```diff
@@ -38,6 +38,26 @@ func sanitizePath(s string) string {
 	return ensureNoTrailingSlash(ensureNoLeadingSlash(s))
 }
 
+// hasTrailingSpace checks to see if there is trailing space
+func hasTrailingSpace(s string) bool {
+	x := strings.TrimSuffix(s, " ")
+	if x != s {
+		return true
+	}
+	return false
```

Comment:
```suggestion
	return x != s
```

Nit: this expression can be simplified